### PR TITLE
[ios] Fix change log typos and re-order

### PR DESCRIPTION
**Pull request by julianrex**
Updates change log and fixes indent issues in log for `5.4.0-alpha.1`

### CODE REVIEW

File: platform/ios/CHANGELOG.md [hunk 1]
```diff
@@ -4,23 +4,29 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
...
+* Fixed an issue that caused the tilt gesture to trigger too easily and conflict with pinch or pan gestures. ([#15349](https://github.com/mapbox/mapbox-gl-native/pull/15349))
 * Fixed an issue that caused the map to rotate too easily during a pinch gesture. [(#15562)](https://github.com/mapbox/mapbox-gl-native/pull/15562)
 
+### Performance improvements
+ 
+* Mark used offline region resources in batches. ([#15521](https://github.com/mapbox/mapbox-gl-native/pull/15521))
+
+### Other changes
+
+* Fixed an issue of integer overflow when converting `tileCoordinates` to `LatLon`. ([#15560](https://github.com/mapbox/mapbox-gl-native/pull/15560))
```
Comment (friedbunny) [inline platform/ios/CHANGELOG.md]: This references Core types that aren’t relevant to iOS developers, so we’ll want to make it more generically worded.

```suggestion
* Fixed a potential integer overflow at high zoom levels. ([#15560](https://github.com/mapbox/mapbox-gl-native/pull/15560))
```

File: platform/ios/CHANGELOG.md [hunk 2]
```diff
@@ -4,23 +4,29 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
...
- * Mark used offline region resources in batches. ([#15521](https://github.com/mapbox/mapbox-gl-native/pull/15521))
 
 ### User interaction
 
+* Fixed an issue that caused the tilt gesture to trigger too easily and conflict with pinch or pan gestures. ([#15349](https://github.com/mapbox/mapbox-gl-native/pull/15349))
 * Fixed an issue that caused the map to rotate too easily during a pinch gesture. [(#15562)](https://github.com/mapbox/mapbox-gl-native/pull/15562)
 
+### Performance improvements
+ 
+* Mark used offline region resources in batches. ([#15521](https://github.com/mapbox/mapbox-gl-native/pull/15521))
```
Comment (friedbunny) [inline platform/ios/CHANGELOG.md]:
```suggestion
* Improved offline region download performance by batching certain database activities. ([#15521](https://github.com/mapbox/mapbox-gl-native/pull/15521))
```

File: platform/ios/CHANGELOG.md [hunk 3]
```diff
@@ -4,23 +4,29 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
...
+
+### Other changes
+
+* Fixed an issue of integer overflow when converting `tileCoordinates` to `LatLon`. ([#15560](https://github.com/mapbox/mapbox-gl-native/pull/15560))
+* Fixed a bug with annotation view positions after camera transitions. ([#15122](https://github.com/mapbox/mapbox-gl-native/pull/15122/))
+* Fixed a bug where the completion block passed to `-[MGLMapView flyToCamera:completionHandler:` (and related methods) wouldn't be called. ([#15473](https://github.com/mapbox/mapbox-gl-native/pull/15473))
+
+
```
Comment (friedbunny) [inline platform/ios/CHANGELOG.md]:
```suggestion
```

File: platform/ios/CHANGELOG.md [hunk 4]
```diff
@@ -4,23 +4,29 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
...
 
+### Performance improvements
+ 
+* Mark used offline region resources in batches. ([#15521](https://github.com/mapbox/mapbox-gl-native/pull/15521))
+
+### Other changes
+
+* Fixed an issue of integer overflow when converting `tileCoordinates` to `LatLon`. ([#15560](https://github.com/mapbox/mapbox-gl-native/pull/15560))
+* Fixed a bug with annotation view positions after camera transitions. ([#15122](https://github.com/mapbox/mapbox-gl-native/pull/15122/))
+* Fixed a bug where the completion block passed to `-[MGLMapView flyToCamera:completionHandler:` (and related methods) wouldn't be called. ([#15473](https://github.com/mapbox/mapbox-gl-native/pull/15473))
```
Comment (friedbunny) [inline platform/ios/CHANGELOG.md]:
```suggestion
* Fixed a bug where the completion block passed to `-[MGLMapView flyToCamera:completionHandler:]` (and related methods) wouldn't be called. ([#15473](https://github.com/mapbox/mapbox-gl-native/pull/15473))
```

**Comment by julianrex** [comment]
This PR was broken, probably due to rebasing 😢. Fixing.

**Pull request merged**